packages/web/pages/_app.tsx

```diff
@@ -135,6 +138,14 @@ const MainLayoutWrapper: FunctionComponent<{
     onClose: onCloseLeavingOsmosisToLevana,
   } = useDisclosure();
 
+  useEffect(() => {
+    if (flags.limitOrders && flags._isInitialized) {
+      document.body.classList.add("!bg-osmoverse-1000");
+    } else {
+      document.body.classList.remove("!bg-osmoverse-1000");
+    }
+  }, [flags.limitOrders, flags._isInitialized]);
+
```

Comment:
Could this affect be avoided? If not, lets make sure it's reversible.

Reply:
We don't have access to the feature flag in globals.css, if there's a better solution to adjust  the body background colour I'm happy to take another approach